patchsets/ruby/2.6.5/railsexpress

```diff
@@ -0,0 +1,4 @@
+https://raw.githubusercontent.com/skaes/rvm-patchsets/master/patches/ruby/2.6.5/railsexpress/01-fix-broken-tests-caused-by-ad.patch
```

Comment:
@skaes Hi, thanks for the patches. Unfortunately the links are broken. All patches for 2.6.5 cannot be found (404).

Reply:
Sorry, I forgot to push :-(

Reply:
Kann passieren ;)